runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessCollection.ts

```diff
@@ -43,6 +43,7 @@ const useProcessCollection = () => {
 
     return ({
         currentCollectionId: collectionId,
+        currentCollection: collectionAncestors[0],
```

Comment:
Is it possible that collectionAcestors will be empty array?

Reply:
it does, corrected to return null if array is empty